quests/the_forge.md

```diff
@@ -251,10 +251,10 @@ class pasture (
 ```
 
 Next, edit the `pasture_config.yaml.epp` template. We'll use a conditional
-statement to only include the `:db_uri:` setting if there is a value set for
+statement to only include the `:uri:` setting if there is a value set for
```

Comment:
Yeah, you're right. I was just thinking "delete one side of the underscore" and picked the wrong one.